var/spack/repos/builtin/packages/gpi-2/package.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+import os
+from spack import *
+
+
+def _verbs_dir():
+    """Try to find the directory where the OpenFabrics verbs package is
+    installed. Return None if not found.
+    """
+    try:
+        # Try to locate Verbs by looking for a utility in the path
+        ibv_devices = which("ibv_devices")
+        # Run it (silently) to ensure it works
+        ibv_devices(output=str, error=str)
+        # Get path to executable
+        path = ibv_devices.exe[0]
+        # Remove executable name and "bin" directory
+        path = os.path.dirname(path)
+        path = os.path.dirname(path)
+        # There's usually no "/include" on Unix; use "/usr/include" instead
+        if path == "/":
+            path = "/usr"
+        return path
+    except TypeError:
+        return None
+    except ProcessError:
+        return None
+
+
+class Gpi2(AutotoolsPackage):
+    """GPI-2 is an API for the development of scalable, asynchronous and fault
+    tolerant parallel applications. It implements the GASPI specification
+    (www.gaspi.de)"""
+
+    homepage = "http://www.gpi-site.com"
+    url      = "https://github.com/cc-hpc-itwm/GPI-2/archive/v1.4.0.tar.gz"
+    git      = "https://github.com/cc-hpc-itwm/GPI-2.git"
+
+    version('master', branch='master')
+    version('next', branch='next')
+    version('1.4.0', sha256='3b8ffb45346b2fe56aaa7ba15a515e62f9dff45a28e6a014248e20094bbe50a1')
+
+    variant('mpi', default=False)
+    variant('fortran', default=True)
```

Comment:
These variants need descriptions

Reply:
I have overlooked this one. Thanks for noticing.